app/page.tsx

```diff
@@ -1,97 +1,12 @@
-import Image from 'next/image';
-import { GithubDoticon } from 'doticons/32';
-import styles from './page.module.scss';
+import DotsBackground from '../components/DotsBackground';
 
-export default function Home() {
+function YourPage() {
```

Comment:
Lets keep this as home
```suggestion
export default function Home() {
```